components/twitch_developer_app/app/twitch_developer_app.app.ts

```diff
@@ -1,13 +1,82 @@
 import { defineApp } from "@pipedream/types";
+import { axios } from "@pipedream/platform";
+import crypto from "crypto";
 
 export default defineApp({
   type: "app",
   app: "twitch_developer_app",
-  propDefinitions: {},
+  propDefinitions: {
+    streamerLoginNames: {
+      type: "string[]",
+      label: "Streamer Login Names",
+      description: "Enter the login names of the streamers whose streams you want to watch.",
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _getBaseUrl() {
+      return "https://api.twitch.tv/helix";
+    },
+    _getHeaders() {
+      return {
+        "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
+        "client-id": this.$auth.client_id,
+        "Content-Type": "application/json",
+      };
+    },
+    async _makeRequest(method:string, endpoint:string, options = {}, $ = this) {
+      const config = {
+        method,
+        url: `${this._getBaseUrl()}/${endpoint}`,
+        headers: this._getHeaders(),
+        ...options,
+      };
+      return axios($, config);
+    },
+    async getUsers(login = []) {
+      const endpoint = "users";
+      const params = {
+        login,
+      };
+      return await this._makeRequest("GET", encodeURI(endpoint), {
+        params,
+      });
+    },
+    async createWebHook(type:string, condition = {}, url:string, secretToken:string) {
+      const endpoint = "eventsub/subscriptions";
+      const data = {
+        "type": type,
+        "version": "1",
+        "condition": condition,
+        "transport": {
+          "method": "webhook",
+          "callback": url,
+          "secret": secretToken,
+        },
+      };
+      return await this._makeRequest("POST", endpoint, {
+        data,
+      });
+    },
+    async getWebHooks(url:string) {
+      const endpoint = "eventsub/subscriptions";
+      let allWebHooks = (await this._makeRequest("GET", endpoint)).data;
```

Comment:
```suggestion
      const allWebHooks = (await this._makeRequest("GET", endpoint)).data;
```
Can we use `const` instead of `let` here?